### PR TITLE
Bump to 0.2: use nested chunks

### PR DESCRIPTION
**Pull request by joshmoore**
see: https://github.com/ome/ngff/pull/40

### CODE REVIEW

**Comment by will-moore** [comment]
I don't know how long all the various version numbers will stay in sync, but for now it would be easier if there was a single `VERSION` variable to use in all these places so we don't miss one.

**Comment by will-moore** [comment]
This PR seems to be causing directories in a Plate to be named with lowercase.
Also, the `.zattrs` and `.zgroup` have the `.` missing from their names:

E.g. `$ omero zarr export Plate:251`

Without this PR:
```
$ ls 251.zarr/
A	B	C
```

With this PR:
```
$ ls 251.zarr/
a	b	c	zattrs	zgroup
```

**Comment by joshmoore** [comment]
@will-moore : it's unlikely I'll be getting to this PR today if you want to push any commits.

**Comment by will-moore** [comment]
Ah, OK the naming of `.zattrs .group` is fixed by upgrading `zarr` to `zarr-2.7.0`.
But still seeing lowercase dir names...

File: src/omero_zarr/raw_pixels.py [hunk 1]
```diff
@@ -191,8 +193,9 @@ def plate_to_zarr(plate: omero.gateway._PlateWrapper, args: argparse.Namespace)
     target_dir = args.output
     cache_dir = target_dir if args.cache_numpy else None
     name = os.path.join(target_dir, "%s.zarr" % plate.id)
-    print(f"Exporting to {name}")
-    root = open_group(name, mode="w")
+    store = FSStore(name, key_separator="/", mode="w", auto_mkdir=True)
```
[the remark this answers is not in the document]
Comment (will-moore) [reply src/omero_zarr/raw_pixels.py:196]:
```suggestion
    store = FSStore(name, key_separator="/", mode="w", auto_mkdir=True, normalize_keys=False)
```

**Comment by will-moore** [comment]
Exporting with this current PR: Data visible at https://mystifying-lalande-e12142.netlify.app/?source=https://minio-dev.openmicroscopy.org/idr/idr0001-graml-sysgro/pr59_nested/2551.zarr

**Pull request merged**